ngraph_bridge/ie_basic_engine.cc

```diff
@@ -0,0 +1,79 @@
+/*******************************************************************************
+ * Copyright 2017-2020 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#include "ngraph_bridge/ie_basic_engine.h"
+#include <iostream>
+#include "logging/ngraph_log.h"
+#include "ngraph_bridge/ie_utils.h"
```

Comment:
General guideline followed in the bridge
`standard includes`
`OV`
`nGraph`
`bridge`